README.rst

```diff
@@ -2,19 +2,17 @@
 AWS Chalice
 ===========
 
-.. image:: https://badges.gitter.im/awslabs/chalice.svg
-   :target: https://gitter.im/awslabs/chalice?utm_source=badge&utm_medium=badge
-   :alt: Gitter
-.. image:: https://readthedocs.org/projects/chalice/badge/?version=latest
-   :target: http://aws.github.io/chalice/?badge=latest
-   :alt: Documentation Status
-
-
 .. image:: https://aws.github.io/chalice/_images/chalice-logo-whitespace.png
    :target: https://aws.github.io/chalice/
    :alt: Chalice Logo
 
 
+**This is the v2 branch of the Chalice framework, which contains in-progress
```

Comment:
I think we should use an admonition for this which can be done with the following markdown:
```
> [!IMPORTANT]  
> <add_your_text_here>
```

The rendered markdown looks like:
> [!IMPORTANT]  
> <add_your_text_here>

Reply:
<img width="256" alt="Screenshot 2025-05-13 at 1 51 30 PM" src="https://github.com/user-attachments/assets/7a2877ae-2ed5-430e-ba64-6e3fbe599751" />

🫠
lol follow up PR to convert readme from rst to markdown...

Reply:
Ah, missed that. Sounds good